Y axis updates

speed-vm opened the pull request:
1) Update to percentages rather than per 100,000 for standardised population plots and breakdowns. I have checked that % comparable with non-standardised.
2) Update y axis labels, to 'each month' and within preceding 12 months

speed-vm committed:
Updates to Y axis

1) calculate % for standardised rates and add to both population and stratified plots

2) Update text on y axis labels

speed-vm committed:
Y axis continuation

Typo, just making sure all y axis text is exactly the same

chrisjwood16 commented on analysis/standardised-plots.py:
```diff
@@ -47,15 +46,22 @@ def checkColumnDict(dic, key):
     df = pd.read_csv(OUTPUT_DIR / f"correctedagegroupsmeasures/{med_review}_population_rate_agesexstandardgrouped_corrected_standardised.csv", parse_dates=["date"])
-    plot_measures(df, filename=f"{med_review}_population_rate_standardised", title="", column_to_plot="UK Standard population rate per 100,000", y_label=f"People who received a {med_review_dict[med_review]} per 100,000 UK Standard population", outputfilepath="figures-standardised") #Plot
+    df['percentrate']=df['UK Standard population rate per 100,000']/1000
+    print(df)
```
Remove this print statement

chrisjwood16 commented on analysis/standardised-plots.py:
```diff
@@ -47,15 +46,22 @@ def checkColumnDict(dic, key):
         df[breakdownbycol] = df[breakdownbycol].fillna('missing')
         if (breakdownby == "care_home_type"): 
-            df=binary_care_home_status(df, numerator_col, 'population',valuecolname="UK Standard population rate per 100,000")
+            df=binary_care_home_status(df, numerator_col, 'percentrate')
```
This isn't quite right as you are passing in 'percentrate' as the denominator and it is calculating a new rate as 'value'. But actually this function shouldn't be used as re-calculation undoes the standardisation. But given we are going with standardisation for care home residents easiest thing is to just remove these lines and remove "care_home_type" from the breakdown list.

speed-vm committed:
Remove care homes as breakdown

As per @chrisjwood16 comments
1) Removed care homes as breakdown as will not be used as a age sex standardised plot.
2) Removed print statement

speed-vm commented:
@chrisjwood16 I have removed the print statement and care homes as a breakdown. Action has now run this morning, successfully
